ehrql/debugger.py

```diff
@@ -145,7 +145,11 @@ def related_columns_to_records(columns):
 
 
 def related_patient_columns_to_records(columns):
-    for patient_id in columns[0].patient_to_value.keys():
+    patients_all = set()
+    for c in columns:
+        for k in c.patient_to_value.keys():
+            patients_all.add(k)
+    for patient_id in patients_all:
```

Comment:
I think we should probably sort here for consistency, otherwise we'll get the patients in an arbitrary order.

Reply:
```suggestion
    for patient_id in sorted(patients_all):
```